sci-astronomy/wcstools/wcstools-3.9.7.ebuild

```diff
@@ -1,13 +1,13 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-inherit autotools
+inherit autotools flag-o-matic
 
 DESCRIPTION="World Coordinate System library for astronomical FITS images"
 HOMEPAGE="http://tdc-www.harvard.edu/software/wcstools/"
-SRC_URI="http://tdc-www.harvard.edu/software/wcstools/Old/${P}.tar.gz"
```

Comment:
You may want to add _two_ SRC_URI entries with Old/ as a fallback. It's likely that old releases get moved there.